ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/AwsExplorer.kt

```diff
@@ -5,44 +5,77 @@ package software.aws.toolkits.jetbrains.uitests.fixtures
 
 import com.intellij.remoterobot.RemoteRobot
 import com.intellij.remoterobot.data.RemoteComponent
+import com.intellij.remoterobot.fixtures.CommonContainerFixture
 import com.intellij.remoterobot.fixtures.FixtureName
 import com.intellij.remoterobot.search.locators.byXpath
 import com.intellij.remoterobot.stepsProcessing.step
+import org.assertj.swing.timing.Pause
 import java.time.Duration
 
 fun RemoteRobot.awsExplorer(
     timeout: Duration = Duration.ofSeconds(20),
     function: AwsExplorer.() -> Unit
 ) {
     step("AWS explorer") {
-        find<AwsExplorer>(byXpath("//div[@class='ExplorerToolWindow']"), timeout).apply(function)
+        find<AwsExplorer>(byXpath("//div[@accessiblename='AWS Explorer Tool Window' and @class='InternalDecorator']"), timeout).apply(function)
```

Comment:
This changed due to it needs to contain both the content (ExplorerToolWindow) and the action stripe so we can find our action buttons